### ISSUE TITLE
Plugin version 3.1.6 issue

### ISSUE THREAD

**Issue by YakobovLior** [logstash-plugins/logstash-output-mongodb#65]
Hello,
I have upgraded the plugin from 3.1.5 to 3.1.6 to have the support of SSL against the MongoDB,
but it looks like version 3.1.6 is accidentally missing the "isodate" option (when I add it to the configuration file it complains of invalid number of arguments, and removing it fixes the issue).
Are there plans to release new version soon which will solve this issue?
Thanks,
Lior

**Comment by rmmorrison** [comment]
I've opened pull request #66 with a fix for this issue.

**Comment by bettersandeep** [comment]
Hey @rmmorrison what else do I need to do in order to replicate your solution on `3.1.6` (locally)? After editing the respective functions in the files, do I need to compile the plugin or anything?

**Comment by rmmorrison** [comment]
@singhksandeep25 yes, you'll need to compile using the instructions in the readme. Once you've got the gem built, it can be added to a Logstash installation using the `logstash-plugin` command, see instructions here: https://www.elastic.co/guide/en/logstash/current/working-with-plugins.html#installing-local-plugins

**Comment by bettersandeep** [comment]
@rmmorrison Thanks a lot!! First I tried to use `logstash-output-mongodb-3.1.5`, but the mongo driver in that version is very old and it was causing troubles while connecting to mongodb atlas. So had to change the dependencies of mongo and bson in the `logstash-output-mongodb.gemspec` and in corporate your changes, the it worked.

**Comment by arichiardi** [comment]
Additional data point on our side, reverting to `3.1.5` worked around the issue
